src/features/toolbar/components/ToolbarControl/ToolbarControl.tsx

```
@@ -43,13 +46,17 @@ export function ToolbarControl({
   switch (settings.type) {
     case 'button':
       const icon = resolveValue(settings.typeSettings.icon, state);
+      const size = isMobile
+        ? (settings.typeSettings.mobilePreferredSize ?? 'medium')
+        : settings.typeSettings.preferredSize;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)